src/Faker/Provider/es_BO/Person.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Faker\Provider\es_BO;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}'
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}'
+    );
+
+    protected static $firstNameMale = array(
```

Comment:
Please mention a valid and MIT compatible datasource in the phpDoc

Reply:
Sorry, what is a valid mit datasource? I'm new in this. I've seen other providers and all those looks like mine.

Reply:
Done